civismlext/test/test_hyperband.py

```diff
@@ -142,41 +142,22 @@ def test_smoke_hyperband(min_iter):
 
 def check_cv_results_array_types(cv_results, param_keys, score_keys):
     # Check if the search `cv_results`'s array are of correct types
-    assert_true(all(isinstance(cv_results[param], np.ma.MaskedArray)
-                    for param in param_keys))
-    assert_true(all(cv_results[key].dtype == object for key in param_keys))
-    assert_false(any(isinstance(cv_results[key], np.ma.MaskedArray)
-                     for key in score_keys))
-    assert_true(all(cv_results[key].dtype == np.float64
-                    for key in score_keys if not key.startswith('rank')))
-    assert_true(cv_results['rank_test_score'].dtype == np.int32)
+    assert (all(isinstance(cv_results[param], np.ma.MaskedArray)
+                for param in param_keys))
+    assert (all(cv_results[key].dtype == object for key in param_keys))
+    assert (any(isinstance(cv_results[key], np.ma.MaskedArray)
+                for key in score_keys)) is False
+    assert (all(cv_results[key].dtype == np.float64
+                for key in score_keys if not key.startswith('rank')))
+    assert (cv_results['rank_test_score'].dtype == np.int32)
 
 
 def check_cv_results_keys(cv_results, param_keys, score_keys, n_cand):
     # Test the search.cv_results_ contains all the required results
     assert_array_equal(sorted(cv_results.keys()),
                        sorted(param_keys + score_keys + ('params',)))
-    assert_true(all(cv_results[key].shape == (n_cand,)
-                    for key in param_keys + score_keys))
-
-
-def check_cv_results_grid_scores_consistency(search):
-    # TODO Remove for sklearn 0.20
```

Comment:
this is removed because `grid_scores_` was replaced by `cv_results_`, right?

Reply:
Yep!